NK_C_API.cc

```diff
@@ -336,6 +336,38 @@ extern "C" {
 		});
 	}
 
+	NK_C_API uint8_t NK_get_pws_slot_count() {
```

Comment:
Let's use uint16_t instead of uint8_t. I am considering using size_t everywhere, since this is internal communication only anyway. What do you think?

Reply:
I thought about that too.  But `NK_get_password_safe_slot_name` etc. already use `uint8_t`, so I think it is better to use it here too.  (Or change both, but that would be a breaking change.)

Reply:
So what you propose is to change them both in sync with the next major version? That might be suiting better indeed from the consistency perspective.